pkg/ui/react-app/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, memo } from 'react';
+import { FormGroup, Label, Input, InputProps } from 'reactstrap';
+
+export interface DropdownOption {
+  name: string;
+  value: string;
+}
+
+interface DropdownProps extends InputProps {
+  options: DropdownOption[];
+}
+
+const Dropdown: FC<DropdownProps> = ({ children, selected, options, id, ...rest }) => {
```

Comment:
Actually, do we need a completely separate component just for single select input? 🙂 

Maybe it would just be easier to add this `<Input>` to Panel? We have an example of this in GraphControl https://github.com/thanos-io/thanos/blob/96fd6db6942290e1e520d556eed2e89df4e29eea/pkg/ui/react-app/src/pages/graph/GraphControls.tsx#L158